chopper_generator/pubspec.yaml

```diff
@@ -1,29 +1,29 @@
 name: chopper_generator
-description: Chopper is an http client generator using source_gen and inspired from retrofit
-version: 3.0.6
-homepage: https://hadrien-lejard.gitbook.io/chopper
+description: Chopper is an http client generator using source_gen, inspired by Retrofit
+version: 4.0.0-nullsafety.0
+documentation: https://hadrien-lejard.gitbook.io/chopper
 repository: https://github.com/lejard-h/chopper
 author: Hadrien Lejard <hadrien.lejard@gmail.com>
 
 environment:
-  sdk: ">=2.2.2 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  meta: ^1.2.3
-  analyzer: ^0.40.4
-  build: ^1.5.0
-  logging: ^0.11.4
-  source_gen: ^0.9.7+1
-  chopper: ^3.0.3
-  code_builder: ^3.5.0
-  dart_style: ^1.3.7
-  built_collection: ^4.3.2
+  analyzer: ^1.2.0
+  build: ^2.0.0
+  built_collection: ^5.0.0
+  chopper: ^4.0.0-nullsafety.0
+  code_builder: ^3.7.0
+  dart_style: ^2.0.0
+  logging: ^1.0.0
+  meta: ^1.3.0
+  source_gen: ^1.0.0
 
 dev_dependencies:
+  pedantic: ^1.11.0
   test: ^1.15.4
-  pedantic: ^1.9.2
 
-# Uncomment for development
-# dependency_overrides:
-#   chopper:
-#     path: ../chopper
+#Comment before publish
+dependency_overrides:
+  chopper:
+    path: ../chopper
```

Comment:
Shouldn't this be commented out again? 😉

Reply:
I can't, it will break the CI since `chopper: 4.0.0...` is not available on pub.dev
And auto publish is not yet implemented with github action, I will publish manually

Reply:
All right, thanks for the explanation! 👍